### PR TITLE
Update CHANGELOG.md for upcoming Django 2.2 version

### PR DESCRIPTION
**Pull request by jechols**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -21,19 +23,27 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
     - This fixes odd issues which can occur when mounting over an existing
       mount point (we mount `.` as `/opt/openoni`, and previously were mounting
       `./docker/data` as `/opt/openoni/data`, effectively shadowing the actual data directory)
-  - All generated ingest artifacts now live in a named volume
+  - All generated ingest artifacts now live in a named volume, `onidata`, which
```
Comment (techgique) [inline CHANGELOG.md]: I think this should note that it only relates to a Docker (Compose) environment.

File: CHANGELOG.md [hunk 2]
```diff
@@ -21,19 +23,27 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
...
 ### Migration
 
 - If you use any of our plugins, make sure you look over their repositories and
-  get a version that is built for Python 3
-- Custom themes and plugins you've built will need to be fixed for Python 3 -
-  exact fixes are out of scope for ONI, but you can learn a lot from the
-  [Python porting documentation](https://docs.python.org/3/howto/pyporting.html)
-  as well as the [2to3](https://docs.python.org/2/library/2to3.html) tool's
-  documentation.
+  get a version that is built for Django 2.2.  Django 2.2 only supports Python
```
Comment (techgique) [inline CHANGELOG.md]: Should this be modified any to tell where to look in the plugin README's etc, now that you've updated those?

File: CHANGELOG.md [hunk 3]
```diff
@@ -21,19 +23,27 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
...
+  3, so plugins which work with Django 2.2 will work with Python 3.
+- Custom themes and plugins you've built will need to be fixed for Django 2.2,
+  and therefore Python 3
+  - Exact fixes are out of scope for ONI, but you can learn a lot from:
+    - The [Python porting documentation](https://docs.python.org/3/howto/pyporting.html)
+    - The [2to3](https://docs.python.org/2/library/2to3.html) tool's documentation
+    - The [Django upgrade documentation](https://docs.djangoproject.com/en/2.2/howto/upgrade-version/)
+  - Most themes and plugins will need minimal work to get updated, but complex
+    Python code could require a lot of migration effort
```
Comment (techgique) [inline CHANGELOG.md]: These seems like an entirely subjective statement that doesn't really help someone migrating :thinking:. They'll find out whether it's hard or easy on their own, I'm thinking.

Comment (jechols) [reply CHANGELOG.md]: It's more of a general warning - if you have a complex plugin, be prepared....

File: CHANGELOG.md (hunk 2 above)
Comment (jechols) [reply CHANGELOG.md]: Agreed.  Gimme a bit and I'll fix that.

Comment (jechols) [reply CHANGELOG.md]: Fixed!

File: CHANGELOG.md (hunk 1 above)
Comment (jechols) [reply CHANGELOG.md]: That's literally the parent of the bullet point in question :)

Comment (techgique) [reply CHANGELOG.md]: Ah okay. I didn't see that as I was just looking at the changes without the full file for context. :+1:

**Pull request merged**